test/integration_test/specs/kubevirt-fedora/portworx/sc-sharedv4svc-nolock.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: storage.k8s.io/v1
```

Comment:
should we remove this one? Why maintain 2 copies of the same thing? It could be confusing in future when you make changes. You have to change the other one not this one.